src/lib/utils-library.ts

```diff
@@ -120,3 +120,10 @@ const SORT_ORDERS: Record<SortBy, SortConfig> = {
 export function getSortOrder(sortBy: SortBy): SortConfig {
   return SORT_ORDERS[sortBy];
 }
+
+/**
+ *  Formats the track's duration from seconds to hh:mm:ss
+ */
+export function formatDuration(seconds: number) {
+  return new Date(seconds * 1000).toISOString().substring(11, 19);
+}
```

Comment:
You can probably re-use that instead, as it's already in the codebase and is unit-tested:

https://github.com/martpie/museeks/blob/01ad926c002db3c11861ed8065aa20ca33b072cb/src/hooks/useFormattedDuration.ts#L3-L24

https://github.com/martpie/museeks/blob/01ad926c002db3c11861ed8065aa20ca33b072cb/src/hooks/__tests__/useFormattedDuration.test.ts#L5-L14

Reply:
Done